pkg/template/static_context.go

```diff
@@ -588,3 +590,20 @@ func (ctx StaticCtx) noProxy() string {
 func (ctx StaticCtx) kotsVersion() string {
 	return strings.TrimPrefix(buildversion.Version(), "v")
 }
+
+func (ctx StaticCtx) yamlEscape(plain string) string {
```

Comment:
Is there a reason you have in mind to use yamlEscape template function instead of using !!binary on the base64encoded data ?

Reply:
Because !! binary is different than base64

Reply:
or at least I thought it was different than base64 - https://yaml.org/type/binary.html

now I feel silly!

Reply:
it seems that that doesn't work great with template functions, so this is still worthwhile 😄

Reply:
soo andrew that answer is what i came up with too :slightly_smiling_face: this works
```
trustStore: !!binary |
          abc=
```
but not `when abc= is replaced with repl{{ ConfigOptionData "custom_trust_store" }}` hence something like this `repl{{ ConfigOptionData "custom_trust_store" | binary }}` would have been ideal, instead of yamlEscape :slightly_smiling_face: but the above is still worthwhile